### ISSUE TITLE
Not able to work 1.5.0 beta 1

### ISSUE THREAD

**Issue by lazyboy0172** [Clooos/Bubble-Card#176]
Installed 1.5.0-beta.1 this morning because I can't leave well-enough alone. 

I believe I followed the instructions, but have an error in my dashboard
Custom element doesn't exist: bubble-card
![image](https://github.com/Clooos/Bubble-Card/assets/57650025/a7976ee7-5462-454c-9d27-84fbc00087d5)

console says I have bubble card v1.5.0-beta.1 installed
![image](https://github.com/Clooos/Bubble-Card/assets/57650025/56c91d18-f5a0-45f5-a933-3b3d0f43e08b)

and I believe I correctly followed the instructions for the config:
![image](https://github.com/Clooos/Bubble-Card/assets/57650025/67e1510d-240e-4612-ace1-cd55a11c8120)

to be sure I didn't have cache issues, I also logged in on a totally new browser that I hadn't been using before, but the issue persisted. Have I made an easy oversight somewhere?

edit:
maybe cache issues, or firefox issue? I re-installed 1.5.0-beta.1 and it worked. went to console and saw that bubble-card.js was loaded twice and giving an error (since reinstalling it loaded it back to lovelace resources). So I deleted from resources again and looked in console again and saw it only once, and still giving issues on firefox. However, on edge where I'd never logged in before, things were working.

If I go to create a new card and search 'bubble', on firefox I have an endlessly-loading item: 
![image](https://github.com/Clooos/Bubble-Card/assets/57650025/8768d382-dbdf-4ae5-88a7-cc5e720eb67b)

whereas on edge it comes up properly:
![image](https://github.com/Clooos/Bubble-Card/assets/57650025/c97b0638-1854-4378-8c1c-44e8781fd4f1)

Clearing all site data on firefox and re-loading, re-logging in etc did not solve it however. I'm still being told element doesn't exist, and trying to add a new card has the forever-loading element where custom:bubble card would be.

finally, as a result of deleting the resource on the front-end, I also get an error in hacs:
![image](https://github.com/Clooos/Bubble-Card/assets/57650025/01a662b2-cc60-4254-b984-d3b9c0a8fced)
I suppose this is to be expected, but it's not ideal to just forever have an error since we're trying to load this through configuration.yaml rather than the dashboard's resources.

**Comment by marcop93** [comment]
Same here. 
Followed all the steps and had the exact same errors as you. Forced the download again and it started working fine right after that, with just a browser force reload (ctrl+F5)

**Comment by Clooos** [comment]
All of this should be fixed in the next beta, I've made a lot more progress since this beta but I still have a bit of work to do before releasing it.

**Comment by Clooos** [comment]
This should be good now : 
https://github.com/Clooos/Bubble-Card/releases/tag/v1.5.0-beta.2
